README.rst

```diff
@@ -29,44 +31,53 @@ sent by any sender.
     This is round two.
     Round 3!
 
-See the [Blinker documentation](https://pythonhosted.org/blinker/) for more information.
+See the `Blinker documentation`_ for more information.
 
-## Requirements
+Requirements
+------------
 
-Blinker requires Python 2.4 or higher, Python 3.0 or higher, or Jython 2.5 or higher.
+Blinker requires Python 2.4 or higher, Python 3.0 or higher, or Jython
+2.5 or higher.
 
-## Changelog Summary
+Changelog Summary
+-----------------
 
 1.3 (July 3, 2013)
 
- - The global signal stash behind blinker.signal() is now backed by a
+-  The global signal stash behind blinker.signal() is now backed by a
    regular name-to-Signal dictionary. Previously, weak references were
    held in the mapping and ephemeral usage in code like
    ``signal('foo').connect(...)`` could have surprising program behavior
    depending on import order of modules.
- - blinker.Namespace is now built on a regular dict. Use
+-  blinker.Namespace is now built on a regular dict. Use
    blinker.WeakNamespace for the older, weak-referencing behavior.
- - Signal.connect('text-sender') uses an alternate hashing strategy to
+-  Signal.connect(‘text-sender’) uses an alternate hashing strategy to
    avoid sharp edges in text identity.
 
 1.2 (October 26, 2011)
 
- - Added Signal.receiver_connected and Signal.receiver_disconnected
+-  Added Signal.receiver\_connected and Signal.receiver\_disconnected
```

Comment:
all this changelog-interpreted-as-markup pain (which will resurface again and again) would go away when not including the changelog into the readme.